app-utils/pmbootstrap/autobuild/build

```diff
@@ -0,0 +1,4 @@
+abinfo "Install pmbootstrap ..."
+
+install -m 755 pmbootstrap.py "$PKGDIR"/usr/bin/pmbootstrap
```

Comment:
Please add verbosity and absolute path per the [Styling Manual](https://wiki.aosc.io/zh/developer/packaging/package-styling-manual/).